app/src/main/java/tenfen/rodolfo/moviesapp/features/home/MovieItem.kt

```diff
@@ -0,0 +1,143 @@
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import java.net.URL
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import tenfen.rodolfo.moviesapp.R
+import tenfen.rodolfo.moviesapp.domain.Genre
+import tenfen.rodolfo.moviesapp.domain.Movie
+import tenfen.rodolfo.moviesapp.ui.theme.listItemBorder
+import tenfen.rodolfo.moviesapp.ui.theme.secondaryText
+
+@Composable
+private fun provideMovieItemViewModel(movie: Movie) = MovieItemViewModel(movie)
+
+val previewMovie =
+    Movie(
+        title = "Title",
+        originalTitle = "Título",
+        backdropUrl = URL("http://example.com/backdrop"),
+        posterUrl = URL("http://example.com/poster"),
+        overview = "This is the overview of the movie. It might be a very long string that will " +
+            "need to be ellipsized: Lorem ipsum dolor sit amet, consectetur adipiscing elit. " +
+            "Mauris magna nunc, fringilla interdum posuere in, tincidunt vel risus.",
+        releaseDate = LocalDate.now(),
+        genres = listOf(Genre(id = 1, "horror"), Genre(id = 2, "comedy"))
+    )
+
+@Preview
+@Composable
+fun MovieItem(movie: Movie = previewMovie) {
+    val viewModel = provideMovieItemViewModel(movie)
+
+    Card(
+        Modifier.padding(horizontal = 8.dp, vertical = 4.dp).wrapContentHeight().fillMaxWidth(),
+        elevation = CardDefaults.cardElevation(2.dp),
+        border = BorderStroke(1.dp, listItemBorder),
+        colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.surface)
+    ) {
+        Row(modifier = Modifier.fillMaxSize(), verticalAlignment = Alignment.Top) {
+            PosterImage(viewModel)
+
+            Column(Modifier.fillMaxHeight().padding(vertical = 8.dp).padding(end = 8.dp)) {
+                Title(viewModel)
+
+                Row(
+                    Modifier.fillMaxWidth().wrapContentHeight().padding(bottom = 8.dp),
+                    verticalAlignment = Alignment.Bottom
+                ) {
+                    OriginalTitle(viewModel)
+
+                    ReleaseDate(viewModel)
+                }
+
+                Text(
+                    text = viewModel.movie.overview,
+                    Modifier.padding(bottom = 8.dp),
+                    overflow = TextOverflow.Ellipsis,
+                    maxLines = 3
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun PosterImage(viewModel: MovieItemViewModel) {
```

Comment:
It might be beneficial to modify this component to accept a parameter such as url: String. This change would enhance its reusability and readability.

Regarding the MoveItemViewModel, I understand the concept; however, I suggest a simpler approach. You could directly utilize the Movie object along with some utility methods. For example, the visibility of the original title could be determined by the following code:

`val isOriginalTitleVisible get() = movie.title != movie.originalTitle`

This approach could streamline the implementation by reducing complexity.

The same apply for the other Components of this screen.